numpy/lib/function_base.py

```diff
@@ -4154,32 +4155,87 @@ def _median(a, axis=None, out=None, overwrite_input=False):
         return mean(part[indexer], axis=axis, out=out)
 
 
+def _valid_quantile(q):
+    # avoid expensive reductions, relevant for arrays with < O(1000) elements
+    if q.ndim == 1 and q.size < 10:
+        for i in range(q.size):
+            if q[i] < 0.0 or q[i] > 1.0:
+                return False
+    else:
+        # faster than any()
+        if np.count_nonzero(q < 0.0) or np.count_nonzero(q > 1.0):
+            return False
+    return True
+
+
 def percentile(a, q, axis=None, out=None,
                overwrite_input=False, interpolation='linear', keepdims=False):
     """
     Compute the qth percentile of the data along the specified axis.
 
-    Returns the qth percentile(s) of the array elements.
+    The ``q``\ th percentile is synonymous with the `quantile` at ``q/100``.
+
+    Parameters
+    ----------
+    q : real number in range of [0,100] (or sequence of real numbers)
+        Percentile to compute, which must be between 0 and 100 inclusive.
+        This is divided by 100 and passed on to `quantile`.
+
+    All the other parameters have the same meaning as they do in `quantile`.
```

Comment:
Don't do that, put them here.

Reply:
If I'm going to put them here, I'd prefer to keep this text in addition, so that someone who just read the percentile docstring knows that the content below is the same, rather than reading it just in case.

Reply:
It's OK for the "Notes" to reference other notes, but I don't think that should be done for the parameters. Again, if we did that everywhere, say for `nansum` and `sum`, so on and so forth, the documetation would become much less useful in interactive use.

Reply:
How do you feel about `percentile` just referencing `quantile`? In interactive use, that will make the user look at and use `quantile` instead, which is likely a good thing.